certified-connectors/tyntec-Viber/apiDefinition.swagger.json

```diff
@@ -343,6 +363,594 @@
           }
         ]
       }
+    },
+    "/conversations/v3/power-automate/messages/viber/text": {
+      "post": {
+        "responses": {
+          "202": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "messageId": {
+                  "type": "string",
+                  "description": "messageId"
+                },
+                "timestamp": {
+                  "type": "string",
+                  "description": "timestamp"
+                }
+              }
+            }
+          }
+        },
+        "operationId": "SendViberTextV3",
+        "summary": "Send Viber Text Message",
```

Comment:
@ridlees we highly suggest you have different description than the summary. can you please change this for all operation in the swagger file? please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

Reply:
Fixed. Did not know it is mandatory now 🌞